src/index.js

```diff
@@ -33,6 +33,7 @@ app.get('*', (req, res) => {
 
 app.use((error, req, res, next) => {
   console.error(error)
+  return res.sendStatus(500)
```

Comment:
What's the purpose of this line?

Reply:
So that the request doesn't hang. I changed it to use the provided `sendDataResponse` function.

Reply:
Ah great! Good catch!